src/main.rs

```diff
@@ -61,6 +60,11 @@ use std::{
 
 type Error = Box<dyn std::error::Error>;
 
+/// Enable showing internal errors via editor UI, such as via hover popups.
+/// This can be helpful for debugging, but we don't want the evaluator to
+/// spam users when confused, so this is disabled by default.
+const DEBUG_TOOLING: bool = false;
```

Comment:
Can we enable this via e.g. a build-flag? I'd like to enable this by default for my dev setu :)

Reply:
Good idea! I've pushed a commit that does this via a feature flag. Thoughts?

Reply:
LGTM.